.github/bin/release-plan/discord-announce.mjs

```diff
@@ -0,0 +1,46 @@
+const defaultPayload = {
+	content: '',
+	embeds: [
+		{
+			title: '',
+			url: '',
+			description: '',
+			color: 3963146,
+		}
+	]
+};
+const BASE_URL = 'https://github.com/csstools/postcss-plugins/tree/main';
+const getChangelog = (changelog) => {
+	const firstUnreleasedIndex = changelog.indexOf('### Unreleased');
+	const firstLineBreakIndex = changelog.indexOf('\n', firstUnreleasedIndex);
+	const secondRelease = changelog.indexOf('###', firstLineBreakIndex);
+	return changelog.slice(firstLineBreakIndex + 1, secondRelease === -1 ? undefined : secondRelease).trim();
+}
+
+export async function discordAnnounce(workspace) {
+	const discordArgument = process.argv.slice(2).find(arg => arg.includes('--discord='));
```

Comment:
This is a bit off but didn't want to add more dependencies or add `.env` handling

Reply:
I agree :)

This is not user facing, so it can be a bit crappy.
It will work great if we use it correctly.